From 81d78b74fc2678c793d96df9a57b0a3e5ca3a470 Mon Sep 17 00:00:00 2001
From: Dominique Leuenberger <dimstar@opensuse.org>
Date: Mon, 11 Apr 2011 22:09:47 +0200
Subject: [PATCH 1/3] Fix the SSH plugin build, bug 647497

Add LIBADD for ssh plugin, and use $(VINAGRE_LIBS) for it. Add $SSH_DEPS
to VINAGRE pkg-config check.
---
 Makefile.am  |    1 +
 configure.ac |    2 +-
 2 files changed, 2 insertions(+), 1 deletions(-)

diff --git a/Makefile.am b/Makefile.am
index 2794c0f..d10a152 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -249,6 +249,7 @@ libssh_la_SOURCES = \
 	plugins/ssh/vinagre-ssh-tab.c
 
 libssh_la_LDFLAGS = $(PLUGIN_LIBTOOL_FLAGS)
+libssh_la_LIBADD = $(VINAGRE_LIBS)
 endif
 
 plugin_LTLIBRARIES = \
diff --git a/configure.ac b/configure.ac
index a169d22..7420fdb 100644
--- a/configure.ac
+++ b/configure.ac
@@ -193,7 +193,7 @@ AS_IF([test "x$have_panelapplet" = "xyes"],
 AM_CONDITIONAL([VINAGRE_HAVE_PANELAPPLET], [test "x$have_panelapplet" = "xyes"])
 
 # Check required libraries
-PKG_CHECK_MODULES([VINAGRE], [$GLIB_DEPS $GTHREAD_DEPS $GTK_DEPS gnome-keyring-1 $XML2_DEPS $PEAS_DEPS $AVAHI_DEPS $TELEPATHY_GLIB_DEPS])
+PKG_CHECK_MODULES([VINAGRE], [$GLIB_DEPS $GTHREAD_DEPS $GTK_DEPS gnome-keyring-1 $XML2_DEPS $PEAS_DEPS $AVAHI_DEPS $TELEPATHY_GLIB_DEPS $SSH_DEPS])
 
 PKG_CHECK_MODULES([VNC], [$GTK_VNC_DEPS $GTK_DEPS $XML2_DEPS])
 
-- 
1.7.4.1

